### ISSUE TITLE
"Unlock with biometrics" button on unlock screen does nothing and occasionally throws errors in the console

### ISSUE THREAD

**Issue by junderw** [bitwarden/clients#12196]
### Steps To Reproduce

1. Install Manjaro Linux Stable
2. Install Brave Browser (brave-browser) from the Official Repositories (Not AUR, flatpak or snap)
3. Install Bitwarden Desktop from Official Repositories (Not AUR, flatpak or snap)
4. Install the Bitwarden Chrome Extension into Brave and log into both Desktop and Extension
5. Use workaround in #11750 to get the extension to pair with the desktop client during the following steps
6. Enable "Unlock with system authentication" in the Desktop client
7. Enable "Unlock with biometrics" and "Ask for biometrics on launch" in browser extension
8. Click user icon, and Lock Now
9. Click on "Unlock with biometrics" while Desktop client is unlocked.

### Expected Result

Some system prompt to ask for the Yubikey PIN and tap the Yubikey (just like Linux login). (This is how it works to unlock the desktop client when it's locked.)

### Actual Result

No response from the UI. The button doesn't seem to respond at all.

Sometimes it outputs errors in the Console of the extension, sometimes it does nothing, and the errors appear after I unlock using normal PIN (unlock with PIN feature).

It seems like there is an attempt to communicate with the desktop client but the packets aren't being answered.

These are the errors that appear most often:

```
chrome-extension://nngceckbapebfimnlniiiahkandclblb/popup/index.html?uilocation=popout#/lock:1 Unchecked runtime.lastError: A listener indicated an asynchronous response by returning true, but the message channel closed before a response was received

console-log.service.ts:51 Unknown error: TypeError: Cannot read properties of undefined (reading 'result')
write @ console-log.service.ts:51
error @ console-log.service.ts:28
(anonymous) @ lock.component.ts:168
a @ main.js:352
invoke @ zone.js:369
onInvoke @ ng_zone.ts:440
invoke @ zone.js:368
run @ zone.js:111
(anonymous) @ zone.js:2538
invokeTask @ zone.js:402
onInvokeTask @ ng_zone.ts:425
invokeTask @ zone.js:401
runTask @ zone.js:159
y @ zone.js:581
Promise.then
d @ zone.js:557
g @ zone.js:568
scheduleTask @ zone.js:391
onScheduleTask @ zone.js:271
scheduleTask @ zone.js:382
scheduleTask @ zone.js:205
scheduleMicroTask @ zone.js:225
j @ zone.js:2528
I @ zone.js:2462
(anonymous) @ zone.js:2370
(anonymous) @ zone.js:2386

Unhandled error in angular TypeError: Cannot read properties of undefined (reading 'filter')
    at Aie.<anonymous> (cipher.service.ts:498:20)
    at Generator.next (<anonymous>)
    at o (main.js:400:545800)
    at c.invoke (zone.js:369:28)
    at Object.onInvoke (ng_zone.ts:440:29)
    at c.invoke (zone.js:368:34)
    at i.run (zone.js:111:43)
    at zone.js:2538:40
    at c.invokeTask (zone.js:402:33)
    at Object.onInvokeTask (ng_zone.ts:425:29)
```

### Operating System

Linux

### Operating System Version

Manjaro Stable (XFCE flavor)

### Web Browser

Brave

### Browser Version

1.73.91-1 (Official Repository's brave-browser)

### Build Version

(Browser Extension) 2024.11.2 ; (Desktop Client) Manjaro Official Repositories (extra) 2024.8.1-1 (Build Date 2024/08/28 01:43:54)

### Issue Tracking Info

- [X] I understand that work is tracked outside of Github. A PR will be linked to this issue should one be opened to address it, but Bitwarden doesn't use fields like "assigned", "milestone", or "project" to track progress.

**Comment by jtodddd** [comment]
Hi there,

I am unable to reproduce this issue, it has been escalated for further investigation. If you have more information that can help us, please add it below. 

Thanks!

**Comment by junderw** [comment]
Might be relevant, but the flavor was XFCE. So I'm still using that...

Is there some way I can see attempts to query for system auth? I could try watching that while I repro the bug and see if I get any logs.

**Comment by junderw** [comment]
I am closing this as an unrelated bug.

I am currently experiencing a bug with AUR's keybase and XFCE, where sometimes apps like thunar (file system) won't open when I click them, but when I close keybase (shutting down kbfs and everything) all the things I did suddenly appear.

The unlock dialogue was the same. Closing keybase made the system auth window show up...

*sighs*

Well at least it's not a Bitwarden problem! lol
